Assets/MixedRealityToolkit.Extensions/Sharing/SpatialAlignment/Common/SpatialCoordinateServiceBase.cs

```diff
@@ -130,19 +135,29 @@ public virtual Task<ISpatialCoordinate> TryCreateCoordinateAsync(Vector3 worldPo
         }
 
         /// <inheritdoc />
-        public virtual Task<bool> TryDeleteCoordinateAsync(string id, CancellationToken cancellationToken)
+        Task<bool> ISpatialCoordinateService.TryDeleteCoordinateAsync(string id, CancellationToken cancellationToken)
         {
             ThrowIfDisposed();
 
-            return Task.FromResult(false);
+            if (!TryParse(id, out TKey key))
+            {
+                throw new ArgumentException($"Id: {id} isn't valid for this spatial coordinate service.");
+            }
```

Comment:
again, its probably worth returning false if this function is expected to potentially fail.

Reply:
Same as the other comment, I think invalid arguments warrants an exception (as a programmer error) compared to an expected failure to find something.